src/moonstone-samples/lib/DatePickerSample.js

```diff
@@ -84,6 +85,7 @@ module.exports = kind({
 			i18n.updateLocale(locale);
 			this.$.picker.setLocale(val);
 			this.$.disabledPicker.setLocale(val);
+			this.$.result.setContent("locale Changed to " + val);
```

Comment:
For consistency and flexibility, can we change "locale" to the value of `event.originator.name`? Also, let's lowercase the "Changed" to "changed".